ArcPicker/ArcPicker.js

```diff
@@ -222,6 +253,7 @@ const ArcPickerBase = kind({
 const ArcPickerDecorator = compose(
 	Changeable,
 	ArcPickerBehaviorDecorator,
+	Spottable,
```

Comment:
![image](https://user-images.githubusercontent.com/5037429/101303173-cced3880-3880-11eb-99c6-bac00e0defb9.png)
Is there a way to only focus on an arc segment or not? It looks so ugly...

Reply:
I fixed as you mentioned so that the Arc could get the accent color instead of ArcPicker itself.